Realm/RLMResults.h

```diff
@@ -408,6 +427,173 @@ __attribute__((warn_unused_result));
                                                          NSError *_Nullable error))block
                                       keyPaths:(nullable NSArray<NSString *> *)keyPaths
 __attribute__((warn_unused_result));
+
+#pragma mark - Flexible Sync
+
+/**
+ Creates an RLMSyncSubscription matching the RLMResults' local filter.
+
+ After committing the subscription to the realm's local subscription set, the method
+ will wait for downloads according to ``RLMWaitForSyncModeOnCreation`` behavior.
+ @see ``RLMWaitForSyncModeOnCreation`` and ``RLMWaitForSyncMode``
+
+ If `subscribeWithCompletion` is called on a query
```

Comment:
Obj-c method names are generally written including the trailing colon as that's a functional part of the name.